Bottom: 817c6f30f8e3616b52468c73f41d9f6a37a6e89e
Top:    ddc946d17210c5130384fb55efeed7143f007760
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-31 01:03:57 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
